aspnetcore/blazor/blazor-server-ef-core.md

```diff
@@ -112,6 +125,23 @@ Finally, [`OnInitializedAsync`](xref:blazor/components/lifecycle) is overridden
 
 [!code-csharp[](./common/samples/5.x/BlazorServerEFCoreSample/BlazorServerDbContextExample/Pages/EditContact.razor?name=snippet2)]
 
+<h3 id="enable-sensitive-data-logging">Enable sensitive data logging</h3>
+
+<xref:Microsoft.EntityFrameworkCore.DbContextOptionsBuilder.EnableSensitiveDataLogging%2A> includes application data in exception messages and framework logging. The logged data can include the values assigned to properties of entity instances and parameter values for commands sent to the database. Logging data with <xref:Microsoft.EntityFrameworkCore.DbContextOptionsBuilder.EnableSensitiveDataLogging%2A> is a **security risk**, as it may expose passwords and other personally identifiable information (PII) when it logs SQL statements executed against the database.
+
+We recommend only enabling <xref:Microsoft.EntityFrameworkCore.DbContextOptionsBuilder.EnableSensitiveDataLogging%2A> with an [`if` compiler directive](/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) for development and testing using the `DEBUG` symbol:
```

Comment:
I'm not sure we want to prescribe this level of detail. I think this may be fine and omit the code snippet:

```suggestion
We recommend only enabling <xref:Microsoft.EntityFrameworkCore.DbContextOptionsBuilder.EnableSensitiveDataLogging%2A> for development and testing.
```